client/src/ui-components/Title.css

```diff
@@ -12,7 +12,7 @@
   position: sticky;
   padding: 2% !important;
   padding-bottom: 0% !important;
-  top: 10%;
+  top: 78.3px !important;
```

Comment:
Setting pixels sizes makes the page not responsive, be careful doing this. Why is this needed?

Reply:
In general, this css class should not exists. It should be a component named Title

Reply:
I used pixel since I want the Row to be exactly at 78.3 px since it's the height of the navbar, using percentage doesn't work (the height of the page may be different based on the size)